buildSrc/src/main/kotlin/plugin-project.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("digma-base")
-    `java`
+    id("common-java")
+    id("common-kotlin")
```

Comment:
maybe add prefix of digma, since common-java sounds some outside infrastructure
```suggestion
    id("digma-common-java")
    id("digma-common-kotlin")
```

Reply:
please open an issue and i will do it